src/common/SConscript

```diff
@@ -58,12 +56,6 @@ def scons():
     base_env.d_add_build_rpath()
     env.d_add_build_rpath()
 
-    # Hack alert, the argobots headers are required but the shared
-    # library isn't so add the dependency so the include path
-    # is correct, but set _libs to [] so that the library isn't
-    # linked in.
```

Comment:
There's something weird going on in this file that this comment relates to.  Just below this there's a `env.require('isal')`, then there's the `denv = env.Clone(LIBS=[])` that this comment refers to and then denv requires 'isal' again.

It would be good to remove the env.require line, and the LIB= from the Clone line to see if this works, then we'll have less code polluting the global env environment.